src/rules/function-url-quotes/index.js

```diff
@@ -12,6 +12,10 @@ export const messages = ruleMessages(ruleName, {
   expected: (q, f="url") => `Expected ${q} around ${f} argument`,
 })
 
+function hasVariable(value) {
+  return value.indexOf("$") !== -1 || value.indexOf("@") !== -1
```

Comment:
If you do `return /\B@/.test(value) || /\B$/.test(value)` does that solve the problem proposed by `url(/images/my_image@2x.png)`?